lib/teleterm/api/proto/v1/service.proto

```diff
@@ -37,8 +37,10 @@ service TerminalService {
     rpc ListDatabases(ListDatabasesRequest) returns (ListDatabasesResponse);
     // ListDatabaseUsers lists allowed users for the given database based on the role set.
     rpc ListDatabaseUsers(ListDatabaseUsersRequest) returns (ListDatabaseUsersResponse);
-    // ListServers lists servers
-    rpc ListServers(ListServersRequest) returns (ListServersResponse);
+    // GetAllServers returns all servers
+    rpc GetAllServers(GetAllServersRequest) returns (GetAllServersResponse);
+    // GetServers returns filtered, sorted, and paginated servers
+    rpc GetServers(GetServersRequest) returns (GetServersResponse);
```

Comment:
Alan introduced some changes to how protos are generated, you might want to merge current master into your branch and run `make grpc-teleterm` again.

Reply:
@codingllama I merged in master to this and i _think_ i didn't upset anything from your changes but could you give a second look to just the proto file to make sure I didn't bork something up please? Thank you!

Reply:
Hey @avatus, could you run `make grpc-teleterm` and push the reformatted files? Without format diffs it'll be easier to spot if we changed something we didn't mean to.

Reply:
whoops, forgot that part 🤦 . thanks. updated now

Reply:
The change looks alright to me - the actual _content_ is up to you guys, but I don't see anything wrong in the proto setup/changes per se.

Word of warning: removing ListServers means that an old client would get an Unimplemented/"unknown RPC" error from a new server. I think this is fine for Connect, because the client and server are always updated together, but it's worth mentioning.

Reply:
Thanks, thats a good thing to keep in mind. Appreciate the 👀